### PR TITLE
update inventory-cli-reference.md incorrect info

### PR DESCRIPTION
**Pull request by irajneeshgupta**
Since the CLI command `inventory:reservation:create-compensations` does not yet accept the `-c`, `-d`, and `-i` argument ,  I rectified few points here for incorrect message string and the arguments.

Correct way to use this command is to pipe both commands like this :
```bash
bin/magento inventory:reservation:list-inconsistencies -r | bin/magento inventory:reservation:create-compensations
```
& if any one need to run it for -i (incomplete orders ) OR -c for complete orders , they may run like by piping and use :
```bash
bin/magento inventory:reservation:list-inconsistencies -r -i | bin/magento inventory:reservation:create-compensations 
``` 
```bash
bin/magento inventory:reservation:list-inconsistencies -r -c | bin/magento inventory:reservation:create-compensations
```
And if anyone wants for all orders , there is no need to specify any argument here like this :

`bin/magento inventory:reservation:list-inconsistencies  | bin/magento inventory:reservation:create-compensations` (For complete orders)

-r  was used to output raw result.
 

## Purpose of this pull request

This pull request (PR) is for rectifying the given page for more clarity to users since those arguments are useless for anyone using 2.3.x or 2.4.x and **the command ``inventory:reservation:create-compensations`` was meant for accepting only STDIN and -r argument only.**

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/inventory/inventory-cli-reference.html

### CODE REVIEW

**Comment by irajneeshgupta** [comment]
#dmcdindia2020

**Review by rogyar — APPROVED** [review]
Hi @irajneeshgupta, thank you for your contribution. 
I can confirm that the mentioned command accepts only `-r` parameter. Example from the codebase below

https://github.com/magento/inventory/blob/1.2-develop/InventoryReservationCli/Command/CreateCompensations.php#L65

**Review by meker12 — CHANGES_REQUESTED** [review]

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

File: src/guides/v2.3/inventory/inventory-cli-reference.md [hunk 1]
```diff
@@ -109,30 +110,44 @@ Following reservations were created:
...
 After updates complete, run the list command to verify:
 
-```terminal
+```bash
 bin/magento inventory:reservation:list-inconsistencies -r
 
 No order inconsistencies were found.
 ```
 
+You can also pipe the commands to detect inconsistencies and create compensations for only incomplete (`-i`) or complete (`-c`) orders.
```
Comment (keharper) [inline src/guides/v2.3/inventory/inventory-cli-reference.md:141]: The line "No order inconsistencies were found." above is output. Separate the output  from the input, as you did for the previous example.

File: src/guides/v2.3/inventory/inventory-cli-reference.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ Command options:
 
 -  `-c`, `--complete-orders` - Returns inconsistencies for completed orders. Incorrect reservations may still be on hold for completed orders.
 -  `-i`, `--incomplete-orders` - Returns inconsistencies for incomplete orders (partially shipped, unshipped). Incorrect reservations may hold too much or not enough salable quantity for the orders.
+-  `-b`, `--bunchsize` - Defines how many orders to load at once.
```
Comment (keharper) [inline src/guides/v2.3/inventory/inventory-cli-reference.md:56]:
```suggestion
-  `-b`, `--bunch-size` - Defines how many orders to load at once.
```

**Review by keharper — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**